src/Adapters/Auth/AuthAdapter.js

```diff
@@ -1,20 +1,102 @@
 /*eslint no-unused-vars: "off"*/
+
+/**
+ * @interface ParseAuthResponse
+ * @property {Boolean} [doNotSave] If true, Parse Server will do not save provided authData.
+ * @property {Object} [response] If set, parse server will send the provided response to the client under authDataResponse
+ * @property {Object} [save] If set, parse server will save the object provided into this key, instead of client provided authData
+ */
+
+/**
+ * AuthPolicy
+ * default: can be combined with ONE additional auth provider if additional configured on user
+ * additional: could be only used with a default policy auth provider
+ * solo: Will ignore ALL additional providers if additional configured on user
+ * @typedef {"default" | "additional" | "solo"} AuthPolicy
+ */
+
 export class AuthAdapter {
-  /*
-  @param appIds: the specified app ids in the configuration
-  @param authData: the client provided authData
-  @param options: additional options
-  @returns a promise that resolves if the applicationId is valid
+  constructor() {
+    /**
+     * Usage policy
+     * @type {AuthPolicy}
+     */
+    this.policy = 'default';
+  }
+  /**
+   * @param appIds The specified app ids in the configuration
```

Comment:
> app IDs

- end lines with dot.
- use backticks for code (vars, methods)
- missing param descriptions

Reply:
```suggestion
   * @param appIds The specified app IDs in the configuration
```